assets/css/v2/dup/departures/row.scss

```diff
@@ -9,7 +11,7 @@
     width: 1586px;
     height: 4px;
     content: "";
-    background-color: #cccbc8;
```

Comment:
There are a lot of places where the colors constants are not used and they should be. Thinking of doing that in a separate quick PR.